api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -513,6 +513,17 @@ public CohortDefinition getPatientsWhoStartedTptAfterPreArt() {
     return cd;
   }
 
+  /**
+   * <b>excluindo os utentes</b>
+   *
+   * <ul>
+   *   <li>com o registo de "Diagnóstico TB Activa (S/ N)" (Coluna 10) = "S" numa consulta clínica
+   *       (Ficha Clínica) que ocorreu no período anterior do relatório e após o início Pré-TARV
+   *       (“Data Consulta” >= “Data Início Pre-TARV” e < “Data Início do Relatório”,
+   * </ul>
+   *
+   * @return
+   */
```

Comment:
This docs mismatch the definition implementation, make sure you are using the correct requirements. If yes, make the necessary changes to match the specs.